sys/src/libthread/BUILD

```diff
@@ -8,7 +8,7 @@ cc_library(
             "//amd64/include",
         ],
 	srcs = [
-		"amd64.c",
+		"$ARCH.c",
```

Comment:
I was hoping you'd ask about this, if it's a `.` or `/` terminated, it doesn't seem like you have to. but I can change this to that form for persistence if you'd like.